### ISSUE TITLE
NegativeSampleGenerator::generateFromRealData: Minimum value must be less than or equal to the maximum value

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/suspicious_login#492]
![Bildschirmfoto von 2021-06-25 08-46-16](https://user-images.githubusercontent.com/1374172/123382129-df675f00-d591-11eb-8ed6-e81e8b1ccc78.png)

cc @adi-dev @GrillGuth @Zocker1999NET seems to be fixable easily with a check for an empty array :)

**Comment by jknockaert** [comment]
Isn't this a duplicate of #443?

**Comment by Knapoc** [comment]
I'm experiencing this bug... are there any temporary fixes like modifying the underlaying data in the database?

**Comment by nunesgh** [comment]
Same error here.

```
$ sudo -u www-data php -f occ suspiciouslogin:train
Using ipv4 strategy
An unhandled exception has been thrown:
Error: Minimum value must be less than or equal to the maximum value in /nextcloud/apps/suspicious_login/lib/Service/NegativeSampleGenerator.php:74
Stack trace:
#0 /nextcloud/apps/suspicious_login/lib/Service/NegativeSampleGenerator.php(74): random_int()
#1 /nextcloud/apps/suspicious_login/lib/Service/NegativeSampleGenerator.php(114): OCA\SuspiciousLogin\Service\NegativeSampleGenerator->generateFromRealData()
#2 [internal function]: OCA\SuspiciousLogin\Service\NegativeSampleGenerator->OCA\SuspiciousLogin\Service\{closure}()
#3 /nextcloud/apps/suspicious_login/lib/Service/NegativeSampleGenerator.php(115): array_map()
#4 /nextcloud/apps/suspicious_login/lib/Service/DataLoader.php(123): OCA\SuspiciousLogin\Service\NegativeSampleGenerator->generateShuffledFromPositiveSamples()
#5 /nextcloud/apps/suspicious_login/lib/Command/Train.php(174): OCA\SuspiciousLogin\Service\DataLoader->generateRandomShuffledData()
#6 /nextcloud/3rdparty/symfony/console/Command/Command.php(255): OCA\SuspiciousLogin\Command\Train->execute()
#7 /nextcloud/3rdparty/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run()
#8 /nextcloud/3rdparty/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand()
#9 /nextcloud/3rdparty/symfony/console/Application.php(149): Symfony\Component\Console\Application->doRun()
#10 /nextcloud/lib/private/Console/Application.php(211): Symfony\Component\Console\Application->run()
#11 /nextcloud/console.php(99): OC\Console\Application->run()
#12 /nextcloud/occ(11): require_once('/nextcloud...')
#13 {main}
```

App version: 4.1.0
Nextcloud version: 23.0.3
PHP version: 7.4.28
MySQL version: 10.5.12

**Comment by joshtrichards** [comment]
Duplicate of #745 
Fixed in #810
